### PR TITLE
Setup CI workflows

### PR DESCRIPTION
**Pull request by tortmayr**
- Adapt publisher name to reflect the eclipse-cdt namespace
- Disable not working test case for now
- Adapt CI workflow
- Add workflow for release

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,8 +2,8 @@
     "name": "vscode-clangd-multi-project",
     "displayName": "clangd-multi-project",
     "description": "C/C++ completion, navigation, and insights",
-    "version": "0.1.26",
-    "publisher": "eclipse-cdt.cloud",
+    "version": "0.1.0",
+    "publisher": "eclipse-cdt",
     "license": "MIT",
     "homepage": "https://www.eclipse.org/cdt-cloud/",
```
Comment (planger) [inline package.json:8]:
```suggestion
    "homepage": "https://eclipse.dev/cdt-cloud",
```

File: package.json [hunk 2]
```diff
@@ -2,8 +2,8 @@
     "name": "vscode-clangd-multi-project",
```
Comment (planger) [inline package.json:2]: Ad the name and display name is something we likely don't want to ever change, maybe we should be more generic here from the start? E.g. `vscode-clangd-cdtcloud`?

**Review by planger — COMMENTED** [review]
Looks good, thank you! A few minor questions below. Before actually releasing we should clarify those. Also I'm wondering whether we should keep the original readme in place, but just add *Fork by Eclipse CDT Cloud* or so in the title and a section what this fork adds to the original VS Code extension?

File: test/inactive-regions.test.ts [hunk 3]
```diff
@@ -1,108 +1,109 @@
...
-  client = new vscodelc.LanguageClient('', {command: ''}, {});
-
-  visibleClangdEditors: vscode.TextEditor[] = [];
-
-  async activate() { throw new Error('Method not implemented.'); }
-
-  dispose() { throw new Error('Method not implemented.'); }
-}
-
-suite('InactiveRegionsFeature', () => {
```
[the remark this answers is not in the document]
Comment (planger) [reply test/inactive-regions.test.ts:22]: I see, understood.

File: package.json [hunk 4]
```diff
@@ -2,8 +2,8 @@
...
     "displayName": "clangd-multi-project",
     "description": "C/C++ completion, navigation, and insights",
-    "version": "0.1.26",
-    "publisher": "eclipse-cdt.cloud",
+    "version": "0.1.0",
```
[the remark this answers is not in the document]
Comment (tortmayr) [reply package.json:5]: If I recall correctly the vscode markeplace does not play well with custom version suffixes.
Versions that have a custom suffix will always be treated as a `pre-release`.
We could use use the same version has the upstream project (0.1.26). However the question then is how do we handle the case when we want to publish a new version but there has not been a new upstream release in the meantime?

[the remark this answers is not in the document]
Comment (planger) [reply package.json]: Ah right, that rings a bell... :-)
Would it be viable to add a build identifier (`0.1.26.1`)?

**Review by planger — APPROVED** [review]

**Pull request merged**